lib/matplotlib/collections.py

```diff
@@ -1106,6 +1107,9 @@ def __init__(self, segments,     # Can be None.
         *zorder*
            The zorder of the LineCollection.  Default is 2
 
+        *facecolors*
+           The facecolors of the LineCollection. Default is 'none'
```

Comment:
Maybe a quick little explanation of what it means for a line collection to have a facecolor?